src/sqlancer/clickhouse/gen/ClickHouseCommon.java

```diff
@@ -0,0 +1,156 @@
+package sqlancer.clickhouse.gen;
+
+import ru.yandex.clickhouse.domain.ClickHouseDataType;
+import sqlancer.IgnoreMeException;
+import sqlancer.Randomly;
+import sqlancer.clickhouse.ClickHouseProvider;
+import sqlancer.clickhouse.ClickHouseSchema;
+import sqlancer.clickhouse.ClickHouseVisitor;
+import sqlancer.clickhouse.ast.*;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+public final class ClickHouseCommon {
```

Comment:
It seems that a number of methods in this class are specific to SQLite. Can we delete those that we do not need for ClickHouse?

Reply:
Sure